### ISSUE TITLE
Just force reload

### ISSUE THREAD

**Issue by somegooser** [grid-js/gridjs#1243]
Hi,

Thanks for this awesome library.

I have read the documentation and found about the forceRender method but im looking for something like forceUpdate.

I just want to fetch data out of the server instead of rendering the table completely. Forcerender also renders my custom plugins and i dont want to that.

I have searched for workarounds like to trigger pagination or sort functions but couldnt find anything.

Thanks!

**Comment by afshinm** [comment]
So you basically want to call the API/Server again and render the rows? i.e a full re-render without rendering the plugins?

**Comment by somegooser** [comment]
Yes i have a button that adds new data to the server. And i would like to tell gridjs to fetch the new data without rendering other parts of the table like search input field or plugins.

**Comment by afshinm** [comment]
That should be fairly easy to implement. I will work on this as soon as #1267 is merged.

**Comment by somegooser** [comment]
That is awesome! Thank you.

**Comment by mnsrulz** [comment]
+1

**Comment by BlackGhost3** [comment]
hello, is this functionality was added ?, because I didn't see it in the documentation. ❤️

**Comment by bruno-buiret** [comment]
Please don't mark this as stale/wontfix =) I look forward to this

**Comment by bruno-buiret** [comment]
Still looking forward to this

**Comment by somegooser** [comment]
@afshinm Hi dude! How is it going? Do you have plans for extending this plugin or doing some maintanance for the current bugfixes? Thank you so much for this awesome plugin.

**Comment by bruno-buiret** [comment]
Please don't mark this as stale :)

**Comment by imanzuk** [comment]
+1

**Comment by bruno-buiret** [comment]
We still don't want it to close :) There's still hope!
